src/hb-ot-cmap-table.hh

```diff
@@ -276,7 +276,9 @@ struct CmapSubtableFormat4
     HBUINT16 *idRangeOffset = serialize_rangeoffset_glyid (c, format4_iter, endCode, startCode, idDelta, segcount);
     if (unlikely (!c->check_success (idRangeOffset))) return;
 
-    if (unlikely (!c->check_assign(this->length, c->length () - table_initpos))) return;
+    if (unlikely (!c->check_assign(this->length,
+                                   c->length () - table_initpos,
+                                   HB_SERIALIZE_ERR_OTHER))) return;
```

Comment:
One of the overflows. I'm trying to separate systemic field overflows, from "something went wrong" while we are at it. Not worth extra effort, but since we're doing this already, maybe okay.